### PR TITLE
Add .Net Core support for log4net

### PR DESCRIPTION
**Pull request by maynardflies**
attempted to preserve all the magic things done in the csproj's to make the references work :)

### CODE REVIEW

File: test/AWS.Logger.Log4Net.FilterTests/Properties/AssemblyInfo.cs [hunk 1]
```diff
@@ -5,7 +5,7 @@
 // General Information about an assembly is controlled through the following 
 // set of attributes. Change these attribute values to modify the information
 // associated with an assembly.
-[assembly: AssemblyTitle("AWS.Logger.Log4Net.FilterTests")]
+//[assembly: AssemblyTitle("AWS.Logger.Log4Net.FilterTests")]
```
Comment (redi-kilicb) [inline test/AWS.Logger.Log4Net.FilterTests/Properties/AssemblyInfo.cs]: Consider removing if not used

File: test/AWS.Logger.Log4Net.FilterTests/Properties/AssemblyInfo.cs [hunk 2]
```diff
@@ -32,5 +32,5 @@
 // You can specify all the values or you can default the Build and Revision Numbers 
 // by using the '*' as shown below:
 // [assembly: AssemblyVersion("1.0.*")]
-[assembly: AssemblyVersion("1.0.0.0")]
-[assembly: AssemblyFileVersion("1.0.0.0")]
+//[assembly: AssemblyVersion("1.0.0.0")]
+//[assembly: AssemblyFileVersion("1.0.0.0")]
```
Comment (redi-kilicb) [inline test/AWS.Logger.Log4Net.FilterTests/Properties/AssemblyInfo.cs]: Same with these commented out lines here

**Comment by spati2** [comment]
Taking a look at the pull request.

**Comment by spati2** [comment]
Thanks for the Pull request. Changes have been merged.

**Pull request merged**